server/controllers/newPlantController.ts

```diff
@@ -0,0 +1,37 @@
+import * as express from 'express';
+import { Request, Response } from 'express';
+import { Plant } from '../models/plant';
+
+const router = express.Router();
+
+router.post('/', async (req: Request, res: Response) => {
+  const newPlant = await Plant.create(req.body);
+  await newPlant.save((err: Error) => {
+    if (err) {
+      return res.status(404).json({ error: 'Plant not saved!' }).end();
+    }
+    return res.status(200).json(newPlant).end();
+  });
+});
+
+router.put('/:id', async (req: Request, res: Response) => {
+  await Plant.findByIdAndUpdate(req.params.id, { accepted: true }, { new: true }, (err: Error) => {
+    if (err) {
+      return res.status(404).json({ error: 'Plant not saved!' }).end();
+    }
+    return res.status(200).end();
+  });
+});
```

Comment:
Move those two to `plantController.ts` and delete the `newPlantController.ts` file.` Delete` endpoint does the same thing in both files so we don't need it.